index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
### Document Title
You should give your file a title name for the document. This will make it easier for the reader to understand what type of file it is on the browser screen.

Reply:
- [ ] For better communication Instead of giving orders, make suggestions. So that the student will be fully motivated to implement the changes you requested. Ex: Instead of saying `You should give your file a title name for the document` just say `I think it would be better to give your file a title name for the document`.